### PR TITLE
Adding gem

### PR DESCRIPTION
**Pull request by volodymyrklymov**
add logic to interact with localize-docs.cirro.io as a gem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Localize::Ruby::Client
+
+TODO: Delete this and the text below, and describe your gem
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/localize/ruby/client`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+## Installation
+
+TODO: Replace `UPDATE_WITH_YOUR_GEM_NAME_IMMEDIATELY_AFTER_RELEASE_TO_RUBYGEMS_ORG` with your gem name right after releasing it to RubyGems.org. Please do not do it earlier due to security reasons. Alternatively, replace this section with instructions to install your gem from git if you don't plan to release to RubyGems.org.
```
Comment (garciaf) [inline README.md:9]: Please update the installation process for new user

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    $ bundle add UPDATE_WITH_YOUR_GEM_NAME_IMMEDIATELY_AFTER_RELEASE_TO_RUBYGEMS_ORG
+
+If bundler is not being used to manage dependencies, install the gem by executing:
+
+    $ gem install UPDATE_WITH_YOUR_GEM_NAME_IMMEDIATELY_AFTER_RELEASE_TO_RUBYGEMS_ORG
+
+## Usage
+
+TODO: Write usage instructions here
```
Comment (garciaf) [inline README.md:21]: Please write the new rake task unlocked and how to configure the gem

File: bin/console [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "localize/ruby/client"
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+require "irb"
```
Comment (garciaf) [inline bin/console:10]: What is it for?

File: lib/localize/ruby/client.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative 'client/version'
+require_relative 'client/translator'
+
+module Localize
+  module Ruby
+    module Client
```
Comment (garciaf) [inline lib/localize/ruby/client.rb:8]: What is this class for?

File: bin/console (hunk 3 above)
Comment (volodymyrklymov) [reply bin/console:10]: I see in Readme "You can also run `bin/console` for an interactive prompt that will allow you to experiment." I think it for checking Gen in console. If it unnecessary, I will remove it

File: lib/localize/ruby/client.rb (hunk 4 above)
Comment (volodymyrklymov) [reply lib/localize/ruby/client.rb]: Fixed to just Client

File: README.md (hunk 1 above)
Comment (volodymyrklymov) [reply README.md]: updated

**Pull request merged**